lix/source-code/fs/src/memoryFs.ts

```diff
@@ -257,11 +311,13 @@ export function createNodeishMemoryFs(): NodeishFilesystem {
 				throw new FilesystemError("ENOENT", path, "rm")
 
 			if (parentDir instanceof Uint8Array) throw new FilesystemError("ENOTDIR", path, "rm")
+			if ("placeholder" in parentDir) throw new FilesystemError("EPLACEHOLDER", path, "readFile")
```

Comment:
"readFile" should be "rm"?
I would fail with "ENOTDIR" here as well? I mean we can add the additional info that it is a placeholder but the code should be "ENOTDIR" because the placeholder is a file - like in the statment before